src/components/job-row.tsx

```diff
@@ -18,6 +18,7 @@ import { IJobParameter, ICreateJobModel } from '../model';
 import { CommandIDs } from '..';
 
 import { outputFormatsForEnvironment } from './output-format-picker';
+import DownloadIcon from '@mui/icons-material/Download';
```

Comment:
Minor: Can you move this import up to line 7 so that it's in the same block as the other imports from `'@mui/icons-material/…'`?